src/main.rs

```diff
@@ -1,22 +1,22 @@
 use crate::options::CLIOptions;
 use clap::{load_yaml, App};
 use rzip::unzip_archive;
+use async_std::{task};
 use std::{fs::File, path::PathBuf};
 mod options;
 
-#[tokio::main]
-async fn main() -> Result<(), Box<dyn std::error::Error>> {
+fn main() {
     let yaml = load_yaml!("../cli_def/en_us.yml");
     let matches = App::from_yaml(yaml).get_matches();
 
     let raw_options = CLIOptions::from_clap_matches(&matches);
 
     let options = raw_options.process_options();
 
-    for filename in options.files.iter() {
-        let file = File::open(filename).unwrap();
-        let _ = unzip_archive(file, &PathBuf::from(&raw_options.destination_folder)).await;
-    }
-
-    Ok(())
+    task::block_on(async {
```

Comment:
I am not sure which is better:
https://docs.rs/async-std/0.99.11/src/async_std/task/block_on.rs.html#41-102
Or `block_on` on the `futures` crate:
https://github.com/rust-lang-nursery/futures-rs/blob/master/futures-executor/src/local_pool.rs#L60

But I just wanted to put it in here. Looks good to me

Reply:
I think they are basically the same. async_std feels like less magic for me, what helps me to reason about it. But at the end I don't see other benefit than just delete the dependency and reducing the compilation time.

Reply:
Makes sense, well if we need it later on we can always change stuff around!